README.md

```diff
@@ -0,0 +1,45 @@
+[![Docker Image Version](https://img.shields.io/docker/v/thedetukov/pgadmin4?sort=date&label=Version)](https://hub.docker.com/r/thedetukov/pgadmin4/tags)
+[![Docker Image Size](https://img.shields.io/docker/image-size/thedetukov/pgadmin4?label=Image%20Size)](https://hub.docker.com/r/thedetukov/pgadmin4/tags)
+[![Docker Pulls](https://img.shields.io/docker/pulls/thedetukov/pgadmin4?label=Pulls)](https://hub.docker.com/r/thedetukov/pgadmin4)
+[![Docker Stars](https://img.shields.io/docker/stars/thedetukov/pgadmin4?label=Docker%20Stars)](https://hub.docker.com/r/thedetukov/pgadmin4)
+
+# pgAdmin
+
+[pgAdmin](https://www.pgadmin.org/) is the most popular and feature rich Open Source administration and development platform for PostgreSQL, the most advanced Open Source database in the world.
+
+# Image reason
+
+No special any reason. This just our infrastructure image.
+
+# Spec
+
+## Environment variables
+
+No any variables
+
+## Expose ports
+
+* `tcp/39000` - pgadmin4 listening endpoint
+
+## Volumes
+
+* `/docker-images` - Root of the pgAdmin repositories
+
+# Inside
+
+* dpage/pgadmin4:latest
```

Comment:
Here we have to define pgAdmin version (not image)

Reply:
Added version to pgAdmin4.